docs/geode-native-docs/serialization/data-serialization.html.md.erb

```diff
@@ -27,3 +27,250 @@ the best combination of versatility and ease-of-use for most applications.
 
 To learn more about other serialization options, see the [Data Serialization section in the _<%=vars.product_name_long%> User Guide_](serverman/developing/data_serialization/chapter_overview.html).
 
+# PdxSerializable Examples
+
+PdxSerializable provides custom serialization to an individual class. Fields within an object can be serialized separately from the rest of the class.
+
+The type of serialization and its implementation can be specified entirely in the client
+application, with no need to create corresponding code on the <%=product_name%> server.
+
+The native client release contains examples, written for .NET and C++, showing how a client application
+can register for serialiation of custom objects using the .NET `IPdxSerializable` or C++ `PdxSerializable` interface.
+
+The examples are located in `examples/dotnet/pdxserializable` and `examples/cpp/pdxserializable`, respectively. 
+
+## Setup
+
+PdxSerializable provides custom serialization to an individual class. Fields within an object can be serialized separately from the rest of the class.
+
+Setting up a class for PdxSerializable treatment requires some preparatory steps:
```

Comment:
I'm not sure I understand how "IPdxSerializable" and "PdxSerializable" are being used here with regard to code formatting/single ticks (i.e., sometimes they are formatted and sometimes not).